src/utils/form-ui.ts

```diff
@@ -87,11 +87,28 @@ function getPredicateName(predicate: string): any {
   return null
 }
 
-async function getPropertyValue(field: string, property: string) {
+/**
+ * Fetch a value for a given subject and property, with special considerations for specific properties
+ * @param field
+ * @param property
+ * @param lang
+ */
+async function getPropertyValue(field: string, property: string, lang: string = 'en') {
   let propertyProxy: any
   const updatedProperty = changeHostProtocol(field)
 
-  if (property.includes('ui#values')) {
+  // For labels, loop over the values of the property. If there are multiple language values they will be returned here in the for await
+  // This is  the recommended way of fetching languages until a more permanent solution is implemented in ldflex
+  if (property === NS.UI.Label) {
+    let labelValue = ''
+    for await (const label of data.from(updatedProperty)[field][property]) {
```

Comment:
Would it add some clarity to the intent of the `for` loop to replace `property` by `NS.UI.Label` ? The result would be equivalent because of the test l.102, and I think it might improve readability a little.

Reply:
Sure I can certainly make that change!